src/DataLevelService.php

```diff
@@ -0,0 +1,312 @@
+<?php
+
+namespace Drupal\commerce_bluesnap;
+
+use Drupal\commerce_store\Entity\StoreInterface;
+use Drupal\commerce_order\Entity\OrderInterface;
+use Drupal\commerce_order\Entity\OrderItemInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * Bluesnap data level service class.
+ *
+ * Provides level 2 and level 3 transaction data.
+ */
+class DataLevelService {
```

Comment:
This should be implementing the DataLevelServiceInterface right?

Reply:
Yep. I had missed that. Added now